### ISSUE TITLE
Support merging reports from android and non-android projects

### ISSUE THREAD

**Issue by ZakTaccardi** [Kotlin/kotlinx-kover#336]
Currently I cannot merge a report from an Android (`kotlin-android`) and a pure JVM (`kotlin`) project.

```kts
dependencies {
  kover(project(":my-android-lib"))
  kover(project(":my-kotlin-lib"))
}
```

```
Could not resolve all task dependencies for configuration ':my-subproject-coverage:koverAggregatedSetup'.
> Could not resolve project :my-android-lib.
  Required by:
      project :my-subproject-coverage
  > The consumer was configured to find attribute 'kotlinx.kover.setup.name' with value ''. However we cannot choose between the following variants of project :my-android-lib:
      - debugApiElements
      - debugRuntimeElements
      - releaseApiElements
      - releaseRuntimeElements
```

This may be a duplicate of #316 - I'm not sure.

**Comment by shanshin** [comment]
Hi,
yes, its duplicate of #316, despite the fact that a multiplatform project was used there

**Comment by shanshin** [comment]
Fixed in 0.7.0-Beta
Please note that the DSL has been modified compared to Alpha version.
